packages/nimble-components/src/table/tests/table.stories.ts

```diff
@@ -125,10 +127,10 @@ const metadata: Meta<TableArgs> = {
             id-field-name="${x => dataSetIdFieldNames[x.data]}"
             data-unused="${x => x.updateData(x)}"
         >
-            <nimble-table-column-text field-name="firstName" placeholder="no value">First Name</nimble-table-column-text>
-            <nimble-table-column-text field-name="lastName" placeholder="no value">Last Name</nimble-table-column-text>
-            <nimble-table-column-text field-name="favoriteColor" placeholder="no value">Favorite Color</nimble-table-column-text>
-            <nimble-table-column-text field-name="quote" placeholder="no value">Quote</nimble-table-column-text>
+            <nimble-table-column-text field-name="firstName" placeholder="no value" column-id="first-name-column">First Name</nimble-table-column-text>
```

Comment:
We should find a place to document the `column-id` attribute and offer guidance about when it is necessary to provide it. The right place would likely be a dedicated column story. I asked about something similar on @atmgrifter00's [column header PR](https://github.com/ni/nimble/pull/1016#discussion_r1092041481) but we haven't decided exactly what we're doing yet. 

Likely the best thing to do is to create an issue to track column APIs that need documentation but I'm open to other ideas or discussion.

Reply:
I've added a note in the description of https://github.com/ni/nimble/issues/1044